### ISSUE TITLE
Update Chinese (Taiwanese) translations

### ISSUE THREAD

**Issue by akiicat** [squidfunk/mkdocs-material#5105]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "zh-TW",
  "action.edit": "編輯此頁",
  "action.skip": "跳轉到",
  "action.view": "查看此頁原始碼",
  "announce.dismiss": "不再顯示此訊息",
  "blog.archive": "封存",
  "blog.categories": "分類",
  "blog.categories.in": "於",
  "blog.continue": "繼續閱讀",
  "blog.draft": "草稿",
  "blog.index": "回到主頁",
  "blog.meta": "元數據",
  "blog.references": "相關連結",
  "clipboard.copy": "複製",
  "clipboard.copied": "已複製",
  "consent.accept": "同意",
  "consent.manage": "管理設定",
  "consent.reject": "拒絕",
  "footer": "頁腳",
  "footer.next": "下一頁",
  "footer.previous": "上一頁",
  "header": "頁首",
  "meta.comments": "留言",
  "meta.source": "來源",
  "nav": "導覽列",
  "readtime.one": "需要 1 分鐘閱讀時間",
  "readtime.other": "需要 # 分鐘閱讀時間",
  "rss.created": "RSS 訂閱",
  "rss.updated": "RSS 訂閱內容已更新",
  "search": "搜尋",
  "search.placeholder": "搜尋",
  "search.share": "分享",
  "search.reset": "清除",
  "search.result.initializer": "正在初始化搜尋引擎",
  "search.result.placeholder": "打字進行搜尋",
  "search.result.none": "沒有符合的項目",
  "search.result.one": "找到 1 個符合的項目",
  "search.result.other": "找到 # 個符合的項目",
  "search.result.more.one": "此頁尚有 1 個符合的項目",
  "search.result.more.other": "此頁尚有 # 個符合的項目",
  "search.result.term.missing": "缺少字詞",
  "select.language": "選擇語言",
  "select.version": "選擇版本",
  "source": "前往倉庫",
  "source.file.contributors": "貢獻者",
  "source.file.date.created": "建立日期",
  "source.file.date.updated": "最後更新",
  "tabs": "標籤",
  "toc": "目錄",
  "top": "回到頂端"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to my best knowledge.
- [X] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks! This is the wrong language code, however!? [Taiwanese is here](https://github.com/squidfunk/mkdocs-material/issues/new?template=04-add-a-translation.yml&title=Update+Chinese+%28Taiwanese%29+translations&translations=%7B%25+macro+t%28key%29+%25%7D%7B%7B+%7B%0A++%22language%22%3A+%22zh-Hant%22%2C%0A++%22action.edit%22%3A+%22%E7%B7%A8%E8%BC%AF%E6%AD%A4%E9%A0%81%22%2C%0A++%22action.skip%22%3A+%22%E8%B7%B3%E8%BD%89%E5%88%B0%22%2C%0A++%22action.view%22%3A+%22View+source+of+this+page+%E2%AC%85%EF%B8%8F%22%2C%0A++%22announce.dismiss%22%3A+%22%E4%B8%8D%E5%86%8D%E9%A1%AF%E7%A4%BA%E6%AD%A4%E8%A8%8A%E6%81%AF%22%2C%0A++%22blog.archive%22%3A+%22Archive+%E2%AC%85%EF%B8%8F%22%2C%0A++%22blog.categories%22%3A+%22Categories+%E2%AC%85%EF%B8%8F%22%2C%0A++%22blog.categories.in%22%3A+%22in+%E2%AC%85%EF%B8%8F%22%2C%0A++%22blog.continue%22%3A+%22Continue+reading+%E2%AC%85%EF%B8%8F%22%2C%0A++%22blog.draft%22%3A+%22Draft+%E2%AC%85%EF%B8%8F%22%2C%0A++%22blog.index%22%3A+%22Back+to+index+%E2%AC%85%EF%B8%8F%22%2C%0A++%22blog.meta%22%3A+%22Metadata+%E2%AC%85%EF%B8%8F%22%2C%0A++%22blog.references%22%3A+%22Related+links+%E2%AC%85%EF%B8%8F%22%2C%0A++%22clipboard.copy%22%3A+%22%E8%A4%87%E8%A3%BD%22%2C%0A++%22clipboard.copied%22%3A+%22%E5%B7%B2%E8%A4%87%E8%A3%BD%22%2C%0A++%22consent.accept%22%3A+%22%E5%90%8C%E6%84%8F%22%2C%0A++%22consent.manage%22%3A+%22%E7%AE%A1%E7%90%86%E8%A8%AD%E5%AE%9A%22%2C%0A++%22consent.reject%22%3A+%22%E6%8B%92%E7%B5%95%22%2C%0A++%22footer%22%3A+%22Footer+%E2%AC%85%EF%B8%8F%22%2C%0A++%22footer.next%22%3A+%22%E4%B8%8B%E4%B8%80%E9%A0%81%22%2C%0A++%22footer.previous%22%3A+%22%E4%B8%8A%E4%B8%80%E9%A0%81%22%2C%0A++%22header%22%3A+%22Header+%E2%AC%85%EF%B8%8F%22%2C%0A++%22meta.comments%22%3A+%22%E7%95%99%E8%A8%80%22%2C%0A++%22meta.source%22%3A+%22%E4%BE%86%E6%BA%90%22%2C%0A++%22nav%22%3A+%22Navigation+%E2%AC%85%EF%B8%8F%22%2C%0A++%22readtime.one%22%3A+%221+min+read+%E2%AC%85%EF%B8%8F%22%2C%0A++%22readtime.other%22%3A+%22%23+min+read+%E2%AC%85%EF%B8%8F%22%2C%0A++%22rss.created%22%3A+%22RSS+feed+%E2%AC%85%EF%B8%8F%22%2C%0A++%22rss.updated%22%3A+%22RSS+feed+of+updated+content+%E2%AC%85%EF%B8%8F%22%2C%0A++%22search%22%3A+%22Search+%E2%AC%85%EF%B8%8F%22%2C%0A++%22search.placeholder%22%3A+%22%E6%90%9C%E5%B0%8B%22%2C%0A++%22search.share%22%3A+%22Share+%E2%AC%85%EF%B8%8F%22%2C%0A++%22search.reset%22%3A+%22Clear+%E2%AC%85%EF%B8%8F%22%2C%0A++%22search.result.initializer%22%3A+%22%E6%AD%A3%E5%9C%A8%E5%88%9D%E5%A7%8B%E5%8C%96%E6%90%9C%E5%B0%8B%E5%BC%95%E6%93%8E%22%2C%0A++%22search.result.placeholder%22%3A+%22%E6%89%93%E5%AD%97%E9%80%B2%E8%A1%8C%E6%90%9C%E5%B0%8B%22%2C%0A++%22search.result.none%22%3A+%22%E6%B2%92%E6%9C%89%E7%AC%A6%E5%90%88%E7%9A%84%E9%A0%85%E7%9B%AE%22%2C%0A++%22search.result.one%22%3A+%22%E6%89%BE%E5%88%B0+1+%E5%80%8B%E7%AC%A6%E5%90%88%E7%9A%84%E9%A0%85%E7%9B%AE%22%2C%0A++%22search.result.other%22%3A+%22%E6%89%BE%E5%88%B0+%23+%E5%80%8B%E7%AC%A6%E5%90%88%E7%9A%84%E9%A0%85%E7%9B%AE%22%2C%0A++%22search.result.more.one%22%3A+%22%E6%AD%A4%E9%A0%81%E5%B0%9A%E6%9C%89+1+%E5%80%8B%E7%AC%A6%E5%90%88%E7%9A%84%E9%A0%85%E7%9B%AE%22%2C%0A++%22search.result.more.other%22%3A+%22%E6%AD%A4%E9%A0%81%E5%B0%9A%E6%9C%89+%23+%E5%80%8B%E7%AC%A6%E5%90%88%E7%9A%84%E9%A0%85%E7%9B%AE%22%2C%0A++%22search.result.term.missing%22%3A+%22Missing+%E2%AC%85%EF%B8%8F%22%2C%0A++%22select.language%22%3A+%22Select+language+%E2%AC%85%EF%B8%8F%22%2C%0A++%22select.version%22%3A+%22Select+version+%E2%AC%85%EF%B8%8F%22%2C%0A++%22source%22%3A+%22%E5%89%8D%E5%BE%80%E5%80%89%E5%BA%AB%22%2C%0A++%22source.file.contributors%22%3A+%22Contributors+%E2%AC%85%EF%B8%8F%22%2C%0A++%22source.file.date.created%22%3A+%22%E5%BB%BA%E7%AB%8B%E6%97%A5%E6%9C%9F%22%2C%0A++%22source.file.date.updated%22%3A+%22%E6%9C%80%E5%BE%8C%E6%9B%B4%E6%96%B0%22%2C%0A++%22tabs%22%3A+%22Tabs+%E2%AC%85%EF%B8%8F%22%2C%0A++%22toc%22%3A+%22%E7%9B%AE%E9%8C%84%22%2C%0A++%22top%22%3A+%22Back+to+top+%E2%AC%85%EF%B8%8F%22%0A%7D%5Bkey%5D+%7D%7D%7B%25+endmacro+%25%7D).

**Comment by akiicat** [comment]
I found it is duplicated #5097

**Comment by squidfunk** [comment]
Ah nevermind. It's all good. We have `zh-Hant` for Taiwanese as well. We should just change it to `zh-TW`.

**Comment by akiicat** [comment]
> Thanks! ~This is the wrong language code, however!?~ Nevermind, the link seems to include the wrong language code (`zh-Hant` instead of `zh-TW`). Can you confirm that the translations are Taiwanese indeed?

I can confirm. Ok, I will change the `zh-Hant` to `zh-TW` in pull request.
Thank you.

**Comment by squidfunk** [comment]
Resolved in #5106.

**Comment by squidfunk** [comment]
Released as part of [9.1.0](https://github.com/squidfunk/mkdocs-material/releases/tag/9.1.0).
